lib/Controller/EmailController.php

```diff
@@ -134,7 +134,7 @@ public function sendEmailPublicLink(string $recipient,
 		}
 
 		return new JSONResponse([
-			'message' => $this->l10n->t('Successfully sent email to ' . $recipient),
+			'message' => $this->l10n->t('Successfully sent email to' . ' ' . $recipient),
```

Comment:
@Valdnet I think @tcitworld meant something like this (not tested):
```suggestion
			'message' => $this->l10n->t('Successfully sent email to %1$s', $recipient),
```